etc/dbus-serialbattery/bms/daly.py

```diff
@@ -339,76 +339,56 @@ def read_alarm_data(self, ser):
         return True
 
     def read_cells_volts(self, ser):
-        if self.cell_count is not None:
-            buffer = bytearray(self.command_base)
-            buffer[1] = self.command_address[0]  # Always serial 40 or 80
-            buffer[2] = self.command_cell_volts[0]
-            buffer[12] = sum(buffer[:12]) & 0xFF
-
-            # logger.info(f"{bytes(buffer).hex()}")
-
-            if (int(self.cell_count) % 3) == 0:
-                maxFrame = int(self.cell_count / 3)
-            else:
-                maxFrame = int(self.cell_count / 3) + 1
-            lenFixed = (
-                maxFrame * 13
-            )  # 0xA5, 0x01, 0x95, 0x08 + 1 byte frame + 6 byte data + 1byte reserved + chksum
-
-            cells_volts_data = self.read_serialport_data(
-                ser, buffer, self.LENGTH_POS, 0, lenFixed
-            )
-            if cells_volts_data is False:
-                logger.warning("No data received in read_cells_volts()")
-                return False
+        if self.cell_count is None:
+            return True
+
+        if (int(self.cell_count) % 3) == 0:
```

Comment:
Can you comment for what this is needed?

Reply:
done